### PR TITLE
Expand number unit system to minutes, hours, and days

### PR DESCRIPTION
**Pull request by codeluggage**
Discussed these changes offline - a quick addition of time units.

### CODE REVIEW

File: packages/imba-monarch/src/utils.imba [hunk 1]
```diff
@@ -193,10 +193,9 @@ export const ToImbaMap = {
 	'Ψ': '#'
 	'Γ': ''
 	'α': '@'
-	'ς': ''
 }
 
-const toImbaRegex = new RegExp("[ΞΦΨΓας]","gu")
+const toImbaRegex = new RegExp("[ΞΦΨΓα]","gu")
```
Comment (codeluggage) [inline packages/imba-monarch/src/utils.imba:198]: Seems like there was partial inclusion of changes in a recent push using a different Greek letter. I believe these should be removed

File: packages/imba/changelog.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
 # Changelog
 
-## 2.0.0-alpha.210
+## 2.0.0-alpha.211
 
-* Add number units for `minutes`, `hours`, and `days`
+- Add number units for `minutes`, `hours`, and `days`
 
-* Add completions for numbered units
+## 2.0.0-alpha.210
```
Comment (codeluggage) [inline packages/imba/changelog.md:7]: Moved the changes up to 211 to account for partial inclusion in `master`.

File: packages/typescript-imba-plugin/src/util.imba [hunk 3]
```diff
@@ -193,10 +193,9 @@ export const ToImbaMap = {
 	'Ψ': '#'
 	'Γ': ''
 	'α': '@'
-	'ς': ''
 }
 
-const toImbaRegex = new RegExp("[ΞΦΨΓας]","gu")
+const toImbaRegex = new RegExp("[ΞΦΨΓα]","gu")
```
Comment (codeluggage) [inline packages/typescript-imba-plugin/src/util.imba:198]: I believe these should be removed.

File: packages/imba/changelog.md (hunk 2 above)
Comment (somebee) [reply packages/imba/changelog.md:7]: We still haven't published 210 yet :) But I can change that after merging.

**Review by somebee — APPROVED** [review]

**Pull request merged**